plugins/rapid7_insightidr/help.md

```diff
@@ -110,7 +110,7 @@ Example output:
 
 #### Advanced Query on Log
   
-Realtime query an InsightIDR log. This will query individual logs for results
+Realtime query an InsightIDR log. This will query individual logs for results. Note a maximum of 500 results only will be returned from a single call, if all results are required for this query please use smaller timeranges
```

Comment:
should 'only' be removed in this sentence? reads a bit strange to me, maybe: 

> Note a maximum of 500 results will be returned from a single call..

or 

> Note only 500 results will be returned from a single call..

Reply:
The wording has been updated to allow the sentance to read better